manim/animation/transform.py

```diff
@@ -238,6 +262,30 @@ def __init__(
 
 
 class CounterclockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Counter-Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: CounterclockwiseTransform1
+        :quality: low
+
+        class CounterclockwiseTransform1(Scene):
+            def construct(self):
+                self.play(CounterclockwiseTransform(Square(), Circle()))
+
+    .. manim:: CounterlockwiseTransform2
```

Comment:
There is a typo here.

```suggestion
    .. manim:: CounterclockwiseTransform2
```